subiquity/server/controllers/filesystem.py

```diff
@@ -491,6 +495,16 @@ async def guided(self, choice: GuidedChoiceV2):
         if gap is None:
             raise Exception('failed to locate gap after adding boot')
 
+        if choice.reset_partition:
+            cp = await arun_command(['du', '-sb', '/cdrom'])
+            reset_size = int(cp.stdout.strip().split()[0])
+            reset_size = align_up(reset_size + GiB, GiB)
```

Comment:
This seems like a lot of padding, potentially 2GiB worth.  Can you elaborate on the reasoning?  I guess there is a sense that the on-disk representation may be physically larger than how it is on the ISO, but this might be too much.

I think it would help to run some tests where we see the implications of staging the live-server ISO in this way, plus a larger one (desktop, ubuntu-studio?) and see how the numbers play out.  Then get a block comment going on the reasoning.

Reply:
I didn't think about this very hard to be honest. It should be aligned to some boundary, obviously, and we should allow a little slop for the reasons you offer. Maybe add 10% and align up to 128M or something?